themes/theme-gmd/pages/Cart/components/ItemsGroup/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Item from '../Item';
+import FulfillmentLocationGroup from './components/FulfillmentLocationGroup';
+
+/**
+ * Renders the product group.
+ * @param {Object} props The component props.
+ * @returns {JSX.Element}
+ */
+const ItemsGroup = ({ group, items, onFocus }) => (
+  <Fragment>
+    {group.fulfillment.locationId &&
```

Comment:
If only `locationId` is needed here, please only input `locationId` as a prop, instead of an object. This is better for performance.